index.js

```diff
@@ -1,33 +1,18 @@
-// Without "./", Node looks
-// in its global module folder
+const nerds = require('nerds')
 const http = require("http");
-function handleRequestAndResponse(req, res) {
-    console.log(req.url);
-    const timestamp = new Date();
-    console.log("Received a Request");
-    res.end(`<h1>Hello World ${timestamp}</h1>`);  // .end() sends the response.
-                                          // It "tapes up the box and ships it".
-}
-// What's a server??!?
-// A server is a piece of code
-// or a computer running a piece of code
-// that does something useful for YOU.
-// Usually, servers store
-// data and often manipulate that data.
-// const server = http.createServer(handleRequestAndResponse);
-const server = http.createServer((req, res) => {
-    console.log(req.url);
-    const timestamp = new Date();
-    console.log("I totally received a Request!!!!!!!!!!!!!!!!!!!");
-    res.end(`<h1>Hello World ${timestamp}</h1>`);  // .end() sends the response.
-                                          // It "tapes up the box and ships it".
-});
 
-// http.createServer() accepts a function and returns a "server"
-// When the server receives a Request:
-// 1. It puts together *most* of a Response.
-// 2. It calls *your* function and passes it the Request and Response
+function createName() {
+    let pokeName = nerds.resolve('Pokemon', 1).include(['name']).asArray()[0].name
+    let swName = nerds.resolve('Star Wars', 1).include(['name']).asArray()[0].name
+    let splitSW = swName.split(' ')
+    if (splitSW.length > 1) {
+        return `${pokeName} ${splitSW.pop()}`
```

Comment:
Interesting! I'm inferring from this line that `pokeName` is only ever a single word, and that `splitSW` sometimes has two values. Consider adding a comment making that clear.